sdk/tests/deqp/framework/opengl/simplereference/sglrReferenceContext.js

```diff
@@ -3805,6 +4326,317 @@ goog.scope(function() {
             this.setError(gl.INVALID_ENUM);
     };
 
+    sglrReferenceContext.ReferenceContext.prototype.texSubImage2D = function(target, level, xoffset, yoffset, width, height, format, type, pixels) {
+        this.texSubImage3D(target, level, xoffset, yoffset, 0, width, height, 1, format, type, pixels);
+    };
+
+    sglrReferenceContext.ReferenceContext.prototype.texSubImage3D = function(target, level, xoffset, yoffset, zoffset, width, height, depth, format, type, pixels) {
```

Comment:
It's pretty weird that texSubImage2D, above, delegates to texSubImage3D. If this is the way it's organized in the original code, that's fine, but otherwise I'd suggest having both delegate to some neutrally-named helper function.